internal/rest/ssl_key_certificate.go

```diff
@@ -152,6 +152,9 @@ func (rest *RestOperations) AviSSLKeyCertAdd(rest_op *utils.RestOp, vsKey avicac
 		case avimodels.SSLKeyAndCertificate:
 			SSLKeyAndCertificate = rest_op.Obj.(avimodels.SSLKeyAndCertificate)
 		}
+		if SSLKeyAndCertificate.Certificate == nil {
```

Comment:
what is the case when we are getting Certificate nil? Is this issue in existing AKO code without your changes?

Reply:
Yes, this has become more prominent in failover scenarios.